codespell_lib/data/dictionary.txt

```diff
@@ -14320,6 +14358,7 @@ programers->programmers
 programing->programming
 programm->program
 programmaticaly->programmatically
+programme->program, programmed, programmer,
```

Comment:
Remove, this is valid BE:
https://dictionary.cambridge.org/dictionary/english/programme